### ISSUE TITLE
no `text_size` in `drawing`

### ISSUE THREAD

**Issue by joseyose** [image-rs/imageproc#486]
Running the font.rs example on the latest crate outputs an error regarding the `text_size` function.

```
error[E0432]: unresolved import `imageproc::drawing::text_size`
 --> src\main.rs:4:41
  |
4 | use imageproc::drawing::{draw_text_mut, text_size};
  |                                         ^^^^^^^^^ no `text_size` in `drawing`
```
```
[dependencies]
imageproc = "0.22.0"
image = { version = "0.23.6", default-features = false }
rusttype = "0.9.2"
```

**Comment by RylanYancey** [comment]
I noticed this as well. Unfortunately it looks like the creator is not going to reply to these issues.

**Comment by joseyose** [comment]
Oh no, is this project not receiving any more updates currently? That's a bummer --- I was looking forward to working with this crate. I guess there's always the tinkering with it myself alternative :)

**Comment by theotherphil** [comment]
I'm the crate maintainer and I'm still around but not actively working on this crate. I'll happily review and accept PRs and can release new versions when it's necessary.

**Comment by joseyose** [comment]
@theotherphil good to hear from you --- that sounds good! I'll see if I can figure out the issue and send a PR. Cheers!

**Comment by tguichaoua** [comment]
The `text_size` function was added on 29 Nov 2020 (see https://github.com/image-rs/imageproc/commit/11b6efac68364b848f42fe3d6d230bbd81d53a75) but `0.22.0` was released on 16 Nov 2020.
This is why the function is not present in the latest version.

**Comment by nyrf** [comment]
Got same error

**Comment by liyongjing** [comment]
image = "0.24.0"
imageproc = { git = "https://github.com/image-rs/imageproc.git" }

**Comment by theotherphil** [comment]
I've just published a new crate version (0.23) that includes @okaneco 's upgrade to image version 0.24.1
